unlock-app/src/__tests__/middlewares/currencyConversionMiddleware.test.js

```diff
@@ -4,8 +4,12 @@ import { setConversionRate } from '../../actions/currencyconvert'
 afterEach(() => mockAxios.reset())
 
 describe('Currency conversion service retrieval middleware', () => {
```

Comment:
Could you remove this file from your commit since these changes are not related to the purpose of your PR? Thanks!

Reply:
This should be fixed once https://github.com/unlock-protocol/unlock/pull/732 has been merged. Could you please rebase after that? Thanks!